GitExtensions/Program.cs

```diff
@@ -100,11 +101,13 @@ private static void RunApplication()
                 }
             }
 
+            bool telemetrySettingChanged = false;
             if (!AppSettings.TelemetryEnabled.HasValue)
             {
                 AppSettings.TelemetryEnabled = MessageBox.Show(null, Strings.TelemetryPermissionMessage,
                                                                Strings.TelemetryPermissionCaption, MessageBoxButtons.YesNo,
                                                                MessageBoxIcon.Question) == DialogResult.Yes;
+                telemetrySettingChanged = true;
```

Comment:
I would save settings here to reduce a chance of showing the same popup to a user if another instance of GE is started before the first instance is closed.